packages/@lwc/engine-dom/scripts/rollup.config.js

```diff
@@ -11,6 +11,7 @@ const path = require('path');
 const { nodeResolve } = require('@rollup/plugin-node-resolve');
 const typescript = require('../../../../scripts/rollup/typescript');
 const writeDistAndTypes = require('../../../../scripts/rollup/writeDistAndTypes');
+const lwcFeatures = require('../../../../scripts/rollup/lwcFeatures');
```

Comment:
Need the same change in `packages/@lwc/engine-server/scripts/rollup.config.js`

Reply:
I didn't bother to make this change for engine-server. I guess I should go ahead and do that.

Reply:
Hm, I could make the change to engine-server, but then the problem is that there's no way to test it. The karma tests don't test engine-server, and the unit tests use the `src` TypeScript files rather than the `dist` files (after running them through Rollup).

I'll just make the change without a test.